src/EnergyPlus/OutdoorAirUnit.cc

```diff
@@ -1215,7 +1207,7 @@ namespace OutdoorAirUnit {
 
         // Do the one time initializations
 
-        auto &OutAirUnit(state.dataOutdoorAirUnit->OutAirUnit);
+        auto &OutAirUnit = state.dataOutdoorAirUnit->OutAirUnit;
```

Comment:
Let's not use constructor initialization notation when simple scalar assignment will do. Also, it messes up my `grep` scripts.